### PR TITLE
Hide bar labels of non highlighted bars

### PR DESCRIPTION
**Pull request by krkshitij**
## Previous Behavior

Before (1)
Before (2)
Before (3)
Before (4)

## New Behavior

After (1)
After (2)
After (3)
After (4)

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/edc52770091144ba9ddb0fd7705d96833fb3b6ff'>edc52770091144ba9ddb0fd7705d96833fb3b6ff</a></sub>
<div id="bundleSizeComment9423"/>

File: packages/react-charting/src/components/StackedBarChart/MultiStackedBarChart.base.tsx [hunk 1]
```diff
@@ -306,7 +310,20 @@ export class MultiStackedBarChartBase extends React.Component<IMultiStackedBarCh
       );
     });
     if (this.props.variant === MultiStackedBarChartVariant.AbsoluteScale) {
-      if (!this.props.hideLabels) {
+      let showLabel = false;
```
Comment (AtishayMsft) [inline packages/react-charting/src/components/StackedBarChart/MultiStackedBarChart.base.tsx:313]: This block of computation can go inside the !this.prop.hideLabels block.

Comment (krkshitij) [reply packages/react-charting/src/components/StackedBarChart/MultiStackedBarChart.base.tsx:313]: done

**Review by AtishayMsft — APPROVED** [review]

File: packages/react-charting/src/components/VerticalStackedBarChart/VerticalStackedBarChart.base.tsx [hunk 2]
```diff
@@ -848,22 +848,35 @@ export class VerticalStackedBarChartBase extends React.Component<
         onClick: this._onClick.bind(this, singleChartData),
         role: 'img',
       };
+      let showLabel = false;
```
[the remark this answers is not in the document]
Comment (krkshitij) [reply packages/react-charting/src/components/VerticalStackedBarChart/VerticalStackedBarChart.base.tsx:851]: done

**Pull request merged**